docs-src/Makefile

```diff
@@ -1,10 +1,24 @@
+L0_VERSION:=$(shell git describe --tags)
+export L0_VERSION_V:=""
+
 deps:
 	pip install mkdocs
 	pip install -Iv mkdocs-material==2.3.0
+	go get github.com/blang/expenv
+	go install github.com/blang/expenv
 
 build:
 	mkdocs build --clean
 
+release:
+	# Remove v from Version
+	$(eval L0_VERSION_V := $(shell echo $(L0_VERSION) | cut -c 2-))
+
+	# Update Version to Latest and clean up
+	expenv -f index.md.template > docs/index.md
```

Comment:
See my last comment -> `README.md` is the **only** place it is ok to use the stripped version, because we know it's specifically for the badge link. We don't want/need a template for README.md, so the `sed` should be done in place.

As I said before, any other replacements need to be done with the **untouched** `L0_VERSION` variable.

Reply:
Okay. Thanks for clarifying I'll fix this.

Reply:
This regex needs to be something like `v[0-9]+.[0-9]+.[0-9]+` 

-> `v<1 to many ints>.<1 to many ints>.<1 to many ints>-` e.g. `v111.2222.33333` is perfectly valid.